### PR TITLE
Fix the onTap callback for images

### LINKED ISSUE #23: Feature: expose MarkdownBuilder for custom Image
I need to add onTap to Image into Markdown.

Currently, it's impossible.

```
class MarkdownBuilder implements md.NodeVisitor {
   Widget _buildImage(String src) {
```

How about this feature?

### PR DESCRIPTION
**Pull request by xqwzts**
Adds onTap handling to images nested in links by wrapping the image in a GestureDetector .

Should fix #23

### CODE REVIEW

File: test/flutter_markdown_test.dart [hunk 1]
```diff
@@ -228,6 +228,22 @@ void main() {
       expect(textSpan.text, 'Hello ');
       expect(textSpan.style, isNotNull);
     });
+
+    testWidgets('should work when nested in a link', (WidgetTester tester) async {
```
Comment (DaveShuckerow) [inline test/flutter_markdown_test.dart]: Add a test with multiple links of different types to be sure that they're populated in-order.

Comment (DaveShuckerow) [reply test/flutter_markdown_test.dart]: Additionally, be sure to test a link that includes an image and text in the same link.

Comment (xqwzts) [reply test/flutter_markdown_test.dart]: Done

**Comment by xqwzts** [comment]
Thanks for the review, I've updated with added tests.

**Pull request merged**